archive/archive_api/src/archive_api.py

```diff
@@ -25,6 +26,11 @@
 daiquiri.setup(level=os.environ.get('LOG_LEVEL', 'INFO'))
 logger = daiquiri.getLogger()
 
+app.config['PROGRESS_MANAGER'] = ProgressManager(
+    endpoint=app.config['PROGRESS_MANAGER_ENDPOINT'],
+    sess=app.config['PROGRESS_MANAGER_SESSION']
+)
+
```

Comment:
This looks odd to me, but perhaps it's a Flask pattern, using the `app.config` for application context?